client/components/Stars.jsx

```diff
@@ -1,12 +1,56 @@
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
-class Stars extends React.Component {
-  render() {
-    return (
-      <div>star1 star2 star3 star4 star5</div>
-    );
+const Container = styled.div`
+  line-height:18px;
+`;
+
+const Svg = styled.svg`
+  display: inline-block;
+  height: 18px;
+  width: 18px;
+  color: ${(props) => props.theme.colors.darkGray};
+  margin: 0 4px 4px 0;
+`;
+
+const stars = {
+  full: <Svg xmlns="http://www.w3.org/2000/svg" viewBox="3 3 18 18" aria-hidden="true" focusable="false"><path d="M20.83,9.15l-6-.52L12.46,3.08h-.92L9.18,8.63l-6,.52L2.89,10l4.55,4L6.08,19.85l.75.55L12,17.3l5.17,3.1.75-.55L16.56,14l4.55-4Z" /></Svg>,
+  half: <Svg xmlns="http://www.w3.org/2000/svg" viewBox="3 3 18 18" aria-hidden="true" focusable="false"><path d="M21.11,10c-.13-.42-.15-.46-.28-.88l-6-.52L12.46,3.08h-.92L9.18,8.63l-6,.52C3,9.57,3,9.61,2.89,10l4.55,4L6.08,19.85l.75.55L12,17.3l5.17,3.1.75-.55L16.56,14ZM12.9,15.79l-.9-.53V6.47l1.21,2.84.41,1,1.05.09,3.07.27-2.32,2-.8.69.24,1,.69,3Z" /></Svg>,
+  empty: <Svg xmlns="http://www.w3.org/2000/svg" viewBox="3 3 18 18" aria-hidden="true" focusable="false"><path d="M12,6.47l1.21,2.84.41,1,1.05.09,3.07.27-2.32,2-.8.69.24,1,.69,3L12.9,15.79l-.9-.53-.9.53L8.45,17.38l.69-3,.24-1-.8-.69-2.32-2,3.07-.27,1.05-.09.41-1L12,6.47m.46-3.39h-.92L9.18,8.63l-6,.52L2.89,10l4.55,4L6.08,19.85l.75.55L12,17.3l5.17,3.1.75-.55L16.56,14l4.55-4-.28-.88-6-.52L12.46,3.08Z" /></Svg>,
+};
+
+function renderRating(rating) {
+  let starNum = rating;
+  const starArray = [];
+  while (starArray.length < 5) {
+    if (starNum >= 1) {
+      starArray.push(stars.full);
+    } else if (starNum >= 0.5) {
+      starArray.push(stars.half);
+    } else {
+      starArray.push(stars.empty);
+    }
+    starNum -= 1;
   }
+  return starArray;
 }
 
+const Stars = (props) => {
+  const { rating } = props;
+  return (
+    <Container>
+      {renderRating(rating)}
+    </Container>
+  );
+};
+
+Stars.defaultProps = {
+  rating: 4.5,
```

Comment:
Why is 4.5 the default rating? Wouldn't the default be 0?

Reply:
Oh that's a god catch, stayed there from a test! Thank you!